### PR TITLE
Issue 2949

### PR DESCRIPTION
**Pull request by meharshahani**
Fixes # 2949

### Description
Removed the numbered list and added a line to make the explanation less redundant.

### Raw.Githack preview link
https://raw.githack.com/meharshahani/meharshahani.github.io/meharshahani-2949/#!pages/manual/planet/overview.md

### CODE REVIEW

File: pages/manual/planet/overview.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ When you are inside PLANET you will see different features available.
 
 ![Planet Dashboard](images/planet-dashboard.png)
 
-List of features:
+<!--List of features:
```
Comment (lmmrssa) [inline pages/manual/planet/overview.md:52]: If you want to remove it then you should remove the code not comment it out.

Comment (meharshahani) [reply pages/manual/planet/overview.md:52]: @lmmrssa Okay I'll do that!! Just wanted to keep it in there until someone approves of it.

Comment (meharshahani) [reply pages/manual/planet/overview.md]: @lmmrssa I made the necessary changes. Please review it.

File: navigation.md [hunk 2]
```diff
@@ -55,7 +55,7 @@
 [Manual]() <!-- System Manual -->
 
   * [Planet User Manual](pages/manual/planet/overview.md)
-  * [myPlanet User Manual](pages/manual/myplanet/overview.md)
+  * [myPlanet User Manual](pages/manual/myplanet/overview.md).
```
[the remark this answers is not in the document]
Comment (lmmrssa) [reply navigation.md:58]: @meharshahani  This change is not needed

[the remark this answers is not in the document]
Comment (meharshahani) [reply navigation.md:58]: @lmmrssa Are you talking about the period in the second line?

**Pull request merged**